app/public/scripts/capyscript.js

```diff
@@ -0,0 +1,26 @@
+const allButtons = document.querySelectorAll("button")
+const allCheckboxes = document.querySelectorAll("input[type=checkbox]")
+
+if(allButtons) {
+    allButtons.forEach(button => {
+        button.addEventListener("click", () => {
+            button.classList.add('clicked');
+            if(button.classList.contains("do-not-click")) {
+                button.innerHTML = "😔 I was clicked against my wishes! 😔";
+            } else {
+                button.innerHTML = "You clicked me!";
+            }
+        });
+    })
+};
+
+if(allCheckboxes) {
```

Comment:
Maybe I'm not understanding what this file does but it doesn't appear to be used ? Once I've run app.rb, I see this script loaded into the instructions page. Once I begin the capybara workout in my irb repl, this script is no longer loaded into my page. 

Since there are no buttons or checkboxes in the instructions, and the ones in the capybara workout page itself use a different script, I wonder what this used for ? 🤔

Reply:
Ah ok it looks like you were using your REPL against this: https://capybara-workout.herokuapp.com/

You need to spin up the site locally, and then use your REPL to interact with https://localhost<port>/workout

Reply:
Ah understood ! I'm on it 😅